### ISSUE TITLE
TypeError: FormatCode() got an unexpected keyword argument 'verify'

### ISSUE THREAD

**Issue by belalhmedan90** [shanice-l/gdrnpp_bop2022#83]
When training Pose Estimation with **hb_pbr** dataset I get an error: `TypeError: FormatCode() got an unexpected keyword argument 'verify'`, full tracing here:

```
20231130_171441|ERR|__main__@233: An error has been caught in function '<module>', process 'MainProcess' (55923), thread 'MainThread' (140702424450880):
Traceback (most recent call last):

> File "./core/gdrn_modeling/main_gdrn.py", line 233, in <module>
    main(args)
    │    └ Namespace(config_file='configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py', dist_u...
    └ <function main at 0x7ff74adbb5e0>

  File "./core/gdrn_modeling/main_gdrn.py", line 199, in main
    Lite(
    └ <class '__main__.Lite'>

  File "/home/belalhmedan/.local/lib/python3.8/site-packages/pytorch_lightning/lite/lite.py", line 408, in _run_impl
    return run_method(*args, **kwargs)
           │           │       └ {}
           │           └ (Namespace(config_file='configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py', dist_...
           └ functools.partial(<bound method LightningLite._run_with_strategy_setup of <__main__.Lite object at 0x7ff7d58c01c0>>, <bound m...
  File "/home/belalhmedan/.local/lib/python3.8/site-packages/pytorch_lightning/lite/lite.py", line 413, in _run_with_strategy_setup
    return run_method(*args, **kwargs)
           │           │       └ {}
           │           └ (Namespace(config_file='configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py', dist_...
           └ <bound method Lite.run of <__main__.Lite object at 0x7ff7d58c01c0>>

  File "./core/gdrn_modeling/main_gdrn.py", line 145, in run
    self.set_my_env(args, cfg)
    │    │          │     └ Config (path: configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py): {'OUTPUT_ROOT':...
    │    │          └ Namespace(config_file='configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py', dist_u...
    │    └ <function Lite.set_my_env at 0x7ff74adbb4c0>
    └ <__main__.Lite object at 0x7ff7d58c01c0>

  File "./core/gdrn_modeling/main_gdrn.py", line 140, in set_my_env
    my_default_setup(cfg, args)  # will set os.environ["PYTHONHASHSEED"]
    │                │    └ Namespace(config_file='configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py', dist_u...
    │                └ Config (path: configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py): {'OUTPUT_ROOT':...
    └ <function my_default_setup at 0x7ff7086ac9d0>

  File "/home/belalhmedan/6D_Pose/gdrnpp_bop2022/core/gdrn_modeling/../../core/utils/default_args_setup.py", line 143, in my_default_setup
    cfg.dump(path)
    │   │    └ 'output/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb/convnext_a6_AugCosyAAEGray_BG05_ml...
    │   └ <function Config.dump at 0x7ff737582ca0>
    └ Config (path: configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py): {'OUTPUT_ROOT':...

  File "/home/belalhmedan/.local/lib/python3.8/site-packages/mmcv/utils/config.py", line 596, in dump
    f.write(self.pretty_text)
    │ │     │    └ <property object at 0x7ff73757d950>
    │ │     └ Config (path: configs/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb.py): {'OUTPUT_ROOT':...
    │ └ <method 'write' of '_io.TextIOWrapper' objects>
    └ <_io.TextIOWrapper name='output/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb/convnext_a...
  File "/home/belalhmedan/.local/lib/python3.8/site-packages/mmcv/utils/config.py", line 508, in pretty_text
    text, _ = FormatCode(text, style_config=yapf_style, verify=True)
    │         │          │                  └ {'based_on_style': 'pep8', 'blank_line_before_nested_class_or_def': True, 'split_before_expression_after_opening_paren': True}
    │         │          └ "OUTPUT_ROOT='output'\nOUTPUT_DIR='output/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb'...
    │         └ <function FormatCode at 0x7ff737571e50>
    └ "OUTPUT_ROOT='output'\nOUTPUT_DIR='output/gdrn/hb_pbr/convnext_a6_AugCosyAAEGray_BG05_mlL1_DMask_amodalClipBox_classAware_hb'...

TypeError: FormatCode() got an unexpected keyword argument 'verify'
```

**Comment by imerino95** [comment]
I solved it installing yapf==0.40.1:

`pip install yapf==0.40.1`

**Comment by belalhmedan90** [comment]
@imerino95 thanks, indeed it solved the problem.
